blog-posts/castore/castore.md

```diff
@@ -0,0 +1,129 @@
+---
+published: false
+title: 'Don’t be a CRUD boomer 👨‍🦳, check out this new Event Sourcing library!'
+description: 'Check out the new best Typescript Event Sourcing library ⚡'
+tags: eventsourcing, typescript, opensource, aws
+series:
+canonical_url:
+---
+
+## Event-sourcing is hard 🔞
+
+Implementing Event Sourcing in Typescript is **painful**. The power of this pattern is often counterbalanced by the **complexity of its implementation** and the **difficulty of its maintainability** 🤯.
+
+<img width="400" src="https://media.giphy.com/media/UETqdVBLO1wMgoy8SE/giphy.gif"/>
+
+At [Kumo](https://dev.to/kumo), many of our projects used Event Sourcing, and we often stumbled upon the same issues again and again and ended up coding the same thing over and over. 🤡 It soon became obvious to us that **developers needed new and better tools to do Event Sourcing on their projects.**
+
+__________________
+
+## We can do better
+
+We thought that the best way to achieve our goal was to **create an open-source library** that would provide the following advantages:
+
+- 🤫 **Concise**: We wanted to **increase the code simplicity** and **provide the optimal developer experience.** Event Sourcing is hard, don't make it harder! 
+- 📝 **Strong typings**: We love type inference and we know you will too!   
+- 👍 **Best practices enforced**: Gained from years of usage at Kumo!  
+- 🛠 **Rich suite of helpers**: Like mock events builder to help you write tests.  
+- 🏄‍♂️ **Stack Agnostic**: We **did not want to enforce any particular implementation** (storage service, messaging system...). Some common implementations are provided, but you are **free to use any implementation you want** via custom classes, as long as they follow the required interfaces. 
+
+__________________
+## Introducing Castore 🦫
+
+As a result, after *months of work ✨*, we are happy to introduce **[Castore](https://github.com/castore-dev/castore)**!
+
+ **Castore is an open-source Typescript library!** It provides a set of classes, methods, and adapters to offer the best typescript developer experience for implementing Event Sourcing and interacting with your database. **V1 of Castore is already live!** 🌟
```

Comment:
`with any storage solution you want to use` ?

Reply:
Done